tracer/src/Datadog.Trace/Ci/Agent/CIWriterFileSender.cs

```diff
@@ -15,7 +15,7 @@ namespace Datadog.Trace.Ci.Agent
     /// <summary>
     /// This class is for debugging purposes only.
     /// </summary>
-    internal sealed class CIWriterFileSender : ICIAgentlessWriterSender
+    internal sealed class CIWriterFileSender : ICIVisibilityProtocolWriterSender
```

Comment:
`CIApp` gave such shorter names 🤣

Reply:
I mean, I used to name classes with `ITR` or `EVP` acronyms, but are reviewer asked me to use the full name 🤣🤣🤣 :trollface: